__mocks__/react-native-safe-area-context.js

```diff
@@ -9,15 +9,17 @@ const insets = {
 };
 
 function withSafeAreaInsets(WrappedComponent) {
-    const WithSafeAreaInsets = (props) => (
-        <WrappedComponent
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...props}
-            // eslint-disable-next-line react/prop-types
-            ref={props.forwardedRef}
-            insets={insets}
-        />
-    );
+    function WithSafeAreaInsets(props) {
```

Comment:
Not sure I understand ... the outer function `withSafeAreaInsets` is a plain function, and the inner on `WithSafeAreaInsets` is a functional component (because it returns a JSX element).

So the inner functional component should follow our functional component style.

Reply:
I can' think of why we'd make an exception here. This change looks good to me.